releasenotes/notes/empty-config-files-924e5ad9c13c1f65.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The Agent will now ignore empty configuration files in `conf.d`. Users can
```

RST uses double backticks for verbatim
```suggestion
    The Agent will now ignore empty configuration files in ``conf.d``. Users can
```